ecr/ref.go

```diff
@@ -30,11 +31,17 @@ import (
 const (
 	refPrefix           = "ecr.aws/"
 	repositoryDelimiter = "/"
+	invalidImageURI = "ECRSpec: Invalid image URI"
```

Comment:
Please start this with "ecrspec:".  See [here](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).